db/seeds.rb

```diff
@@ -8,7 +8,7 @@
 # Set up a default admin user
 u = User.find_or_create_by(email: 'admin@example.com')
 u.display_name = "Default Admin"
-u.password = ENV['ADMIN_PASSWORD']
+u.password = ENV['ADMIN_PASSWORD'] || 'password'
```

Comment:
this is not the documented default admin password, we should revise the docs if this will remain as-is, and we should probably add an example of setting this value in the .env.sample file